docs/planning/bug_triage.md

```diff
@@ -0,0 +1,44 @@
+# Knative Eventing Bug Triage and Management
+
+This document summarizes how bugs will be triaged and managed in Knative Eventing WG.
+This is a supplement to [Sprint planning](sprintplanning.md) process.
+
+## Goal
+
+- Keep a low bug count. Bug count should not be consistently increasing release over release esp [priority/important-longterm](#Bug-priorities).
+- Process shouldn't be taxing for a single person or the team.
+- Maintain high product quality.
+
+## Bug priorities
+
+- priority/critical-urgent: Bugs that need to be fixed now, and possibly cherrypicked into the most recent release. May need patch-fix.
+- priority/important-soon: Bugs that need to be fixed in the next release cycle, and but do not need a patch-fix.
+- priority/important-longterm: Bugs that need to be fixed, and will be fixed in next 2-3 releases.
+- priority/awaiting-more-evidence: Bugs which might need to be fixed, but need more details first.
+
+## How to manage bugs?
+
+- Submit new bugs anytime by creating a new issue of type “Bug”.
+- Please fill the bug template and optionally propose a priority.
+- All new bugs will be labelled “kind/bug” as part of the template
+ Bugs will be triaged weekly (this could be modified based on incoming bugs
+  rate), and upon triage will be assigned a priority or closed.
+- During each release/sprint planning, all priority/important-soon and some of priority/important-longterm bugs need to be accounted while commiting to new features and improvements. These will be marked with approapriate release along with the planned features.
+- Bugs marked with current release always take priority over feature work in a particular release. This will help maintain a balance between code quality, bug triage bar, and incoming feature rate.
+
+
+## Bug Triage
+
+- WG lead will form a bug triage team (TODO: Ville)
+- There will be one bug triage champion each week to triage incoming bugs and assign them correct priority. This role will be rotated among engineers in the bug triage team.
+- We will have first 3-4 bug triage meetings as a weekly team meeting. This meeting will help the entire team to come to an agreement and set a bar for each priority level. WG lead will drive this meeting.
+- Guidelines for bug triage:
+    - Any priority/critical-urgent bug will disrupt the current release cycle and hence need to be approved by WG lead.
+    - All of priority/important-soon bugs will be part of the next release cycle.
+    - Number of bugs with priority/important-longterm need to be kept low and non-increasing, release over release. Here "long term" refers to next 2-3 release cycles. If we will not work on it in next 2-3 release cycles, then we should close the bug upon triaging.  [Remember the goal](#Goal). 
```

Comment:
How is this assessed? Do we have any metrics on this?

If we don't have metrics on this, when will we need metrics? Are there any existing tools for this?

Reply:
maybe this can be manual at the beginning with every release? dumbest idea is to have a spreadsheet with a dumb chart of bugs per release that gets updated with the number of current open bugs during release time. Not sure.

Reply:
I am still looking for a tool. Eric Allred is exploring some of these tools (based on our discussion few weeks back). I'll connect with him figure out how this could be tracked. I didn't define any metrics and threshold to begin with. Initially, I was thinking of just tracking number of triaged and accepted bugs release over release, but still looking for a good tool to do this.